src/syntax-tree/ast.ts

```diff
@@ -2684,7 +2667,7 @@ export class Module {
                             parentStatement.dataType == DataType.Any &&
                             !(
                                 focusedNode.rootNode instanceof ComparatorExpr ||
-                                focusedNode instanceof EmptyListItem ||
+                                // focusedNode instanceof EmptyListItem ||
```

Comment:
I changed this check in my code to fix another issue. What was the issue that caused you to comment it out? I want to make sure that my code does not reintroduce it.

Reply:
`EmptyListItem` just doesn't exist any more :)

Reply:
Perfect :)